s3router.js

```diff
@@ -25,9 +25,9 @@ function S3Router(options) {
 
     /**
      * Redirects image requests with a temporary signed URL, giving access
-     * to GET an image.
+     * to GET an upload.
      */
-    router.get(/\/img\/(.*)/, function(req, res) {
+    router.get(/\/upload\/(.*)/, function(req, res) {
```

Comment:
`uploads` works for me too, I originally did that actually haha.  I'll switch it to that.  Also I'll do figure out the backwards compatibility too.